### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by ZDisket** [dathudeptrai/TensorflowTTS#18]
1. Are the mel outputs generated compatible with kan-bayashi's ParallelWaveGAN?
2. There's a FastSpeech synthesis example, but not Tacotron2. How to generate speech with the Tacotron2 pretrained model and MelGAN-STFT?

**Comment by dathudeptrai** [comment]
Hi,

> Are the mel outputs generated compatible with kan-bayashi's ParallelWaveGAN?
We have different train/valid split. But generally, the preprocessing steps is the same and the mean/var of our training very close. So let say, i believe it's compatible, you can use Tacotron, Fastpeech generated from this repo and use pretrained models from ParallelWaveGAN.

> There's a FastSpeech synthesis example, but not Tacotron2. How to generate speech with the Tacotron2 pretrained model and MelGAN-STFT?.
To know how to inference, you can see detail at decode_tacotron2.py or decoder_melgan.py in examples directory. Melgan-STFT is melgan but training with Multi-resolution STFT loss so it's inference same as Melgan original.

**Comment by dathudeptrai** [comment]
@ZDisket https://github.com/dathudeptrai/TensorflowTTS/blob/master/notebooks/tacotron2_inference.ipynb. Just in case :D

**Comment by ZDisket** [comment]
@dathudeptrai 
Nice. It seems pretty similar to [my notebook](https://colab.research.google.com/drive/1SLl0tVT6tBu1zEn7atlrMWMIk1ucLRd0?usp=sharing).

**Comment by dathudeptrai** [comment]
@ZDisket great :)). I just uploaded Tacotron pretrained 120K. I'm training multiband melgan, it will 3x faster and improve quality compared with melgan-stft :D.

**Comment by dathudeptrai** [comment]
why mb-melgan on kan-bayashi not worked ?

**Comment by ZDisket** [comment]
@dathudeptrai  All my [predictions](https://drive.google.com/drive/folders/1-Ep7YnBKO_YvXfSod9GQmaFu_63-85f9?usp=sharing) had heavy metallic noise, and when it reaches the discriminator train start steps (on another training run) they all become pure noise.

**Comment by dathudeptrai** [comment]
okay, let see how mb melgan on this repo help you. It will finish training progress on saturday, i think :D.

**Comment by ZDisket** [comment]
@dathudeptrai 
I can't see Tacotron2-120k in the pretrained models section in [here](https://github.com/dathudeptrai/TensorflowTTS/tree/master/examples/tacotron2). Where is it?

**Comment by dathudeptrai** [comment]
https://drive.google.com/drive/u/1/folders/1kaPXRdLg9gZrll9KtvH3-feOBMM8sn3_
@ZDisket
